### PR TITLE
Improve header 2

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- Part of a waterfall PR, the final one being #596 
- Minor styling
- Makes sure the whole account section in the header (balance + account address) will trigger the `openOrdersPanel` on click. Before it was only on click of the address. Now the whole pill is clickable.

### CODE REVIEW

**Review by alfetopito — APPROVED** [review]
Changes described here work as intended.

Side note, the styles seems to have changed somewhere along the way and now account balance and account address have the same background
Screen Shot 2022-05-31 at 09 30 37

I assume this will be changed in the follow up PRs
Also, there's no menu as of yet, so I can't change dark mode etc

**Comment by fairlighteth** [comment]
Thanks for reviewing @alfetopito 

> I assume this will be changed in the follow up PRs

This is correct.

File: src/custom/pages/Profile/index.tsx [hunk 1]
```diff
@@ -499,7 +499,7 @@ export default function Profile() {
               </FlexWrap>
             </ChildWrapper>
           </GridWrap>
-          {!account && <Web3Status openOrdersPanel={() => console.log('TODO')} />}
+          {!account && <Web3Status />}
```
Comment (W3stside) [inline src/custom/pages/Profile/index.tsx:502]: be careful with this if there is async logic setting state or whatever inside Web3Status as you'll get memory leaks. Could be best to render conditoinally inside `Web3Status`

**Review by W3stside — APPROVED** [review]

**Comment by nenadV91** [comment]
I'm not sure if this is intended, or if not maybe its fixed in the following PR's but now if I'm not connected and click on the `Connect to a wallet` header button both wallet and orders modals open
Screenshot from 2022-05-31 14-36-51

File: src/custom/pages/Profile/index.tsx (hunk 1 above)
Comment (fairlighteth) [reply src/custom/pages/Profile/index.tsx:502]: When you get to review https://github.com/cowprotocol/cowswap/pull/596 please double check if this is still a concern, and whether it should be fixed.

**Pull request merged**